sunpy/map/mapbase.py

```diff
@@ -752,15 +749,71 @@ def date(self):
             # UTC is the FITS standard default
             timesys = self.meta.get('timesys', 'UTC')
 
+        return parse_time(time, scale=timesys.lower())
+
+    @property
+    def date_beg(self):
+        """
+        Time of the beginning of the image acquisition.
+
+        Taken from the DATE-BEG FITS keyword.
+        """
+        return self._get_date('date-beg')
```

Comment:
I think given the following quote from Section 4.4.2.2 of the FITS 4 spec:

> When the format with a four-digit year is used, [...]. The value of DATE-OBS shall be assumed to refer to the start of an observation, unless another interpretation is clearly explained in the comment field.

We could change this to:

```suggestion
        return self._get_date('date-beg', self._get_date('date-obs'))
```

and remove the special casing from `.date`?

Reply:
If we're not going to check the comment field (which I think is too complicated for us to do and explain in a docstring), then I don't think we can be sure that `DATE-OBS` is the start of the observation, so I'm hesitant make this change.

Reply:
That sounds good to me.